test/witan/send/acceptance/validate_model_test.clj

```diff
@@ -1,9 +1,10 @@
 (ns witan.send.acceptance.validate-model-test
-  (:require [clojure.test :refer :all]
-            [clojure.java.io :as io]
+  (:require [clojure.java.io :as io]
             [clojure.string :refer [join]]
+            [clojure.test :refer [deftest is testing]]
```

Comment:
What does this mean?

Reply:
It just explicitly refers things in rather than a refer all. I'm deciding if I prefer it or not. It does have the advantage of being specific and keeping the joker linter quiet. :smile_cat:

Reply:
I mean `[deftest is testing]`? What does "is testing" mean?

Reply:
`deftest`, `is`, and `testing` are the 3 macros that are being referred to, so there is no `is testing` as such.